### DISCUSSION TITLE
Release 3.40.0?

### DISCUSSION THREAD

**Discussion by seanbright** [asterisk-java/asterisk-java#635] [General]
Hi @piotrooo,

I would like to release 3.40.0. I see that there is a "deploy" profile in the POM and I looked at some of the commit history to figure it out but I was hoping you could give me a quick run-down on how it works?

Thank you!
Sean

**Comment by piotrooo** [comment]
Sure, here is a video tutorial for you :wink: 
[screencast-github.com-2024.08.30-06_49_56.webm](https://github.com/user-attachments/assets/1e4179fa-9f58-4faa-9795-f7a402e66d71)

BTW, give me a couple of days, and I';; come back with ideas for the version 4.0.0 discussion.

**Reply by seanbright** [reply]
I just watched the video. I know how to create releases in GitHub, but I was trying to understand the release process as it relates to nexus, etc. 

I’ll figure it out. Thank you.

**Reply by seanbright** [reply]
FYI - I changed the release that you created to a pre-release because of the version issue. Ideally the POM version would have been 3.40.0 at the tag and then 3.41.0-SNAPSHOT in master/main.

I'll do some testing in a private repository with actions and see if I can get that working the way I want.

**Reply by seanbright** [reply]
I see the issue - the publish action failed:

https://github.com/asterisk-java/asterisk-java/actions/runs/10627013877/job/29459459060

**Comment by seanbright** [comment]
I see that you created the release already but the POM and README versions were not updated beforehand so they still reference 3.39.0.

Was I supposed to update those before you released?

**Comment by seanbright** [comment]
Hi @piotrooo - can you please make me an owner of the organization? If not, can you provide me the nexus credentials so that I can try to debug the release actions?

**Reply by seanbright** [reply]
Ping @piotrooo

**Reply by piotrooo** [reply]
I don't have Nexus credentials. I have been using the ones in the secrets.

**Reply by seanbright** [reply]
@piotrooo please make me an owner of the asterisk-Java organization

**Reply by piotrooo** [reply]
Done, I guess.
![Screenshot from 2024-09-12 14-27-46](https://github.com/user-attachments/assets/c71efed3-42c0-4dd5-a15f-fc3edb2f3977)

**Reply by seanbright** [reply]
Thank you!
